src/util/Util.js

```diff
@@ -307,6 +307,100 @@ export function IsPointInsidePolygon(point, vertices)
 	return isInside;
 }
 
+/**
+ * Return random floats a-la NumPy's in the half-open interval [0.0, 1.0). In other words, from 0 inclusive to 1 exclusive.
+ * 
+ * @param {number} [size = 1] - number of values to return
+ * @returns {number} single random float from uniform distribution, if size === 1
+ * @returns {Object[]} array of uniformly distributed random floats, if size > 1
+ */
+export function random(size = 1) {
+	if (!Number.isInteger(size) | size < 1) {
+		// raise error if given an invalid size
+		throw {
+			origin: "util.random",
+			context: "when generating a random float",
+			error: "size must be a positive integer above 0",
+		};
+	}
+
+	if (size > 1) {
+		// if size > 1, call function multiple times with size = 1 and return an array
+		let values = []
+		for (let i = 0; i < size; i++) {
+			values.push(random(1));
+		}
+		return values
+	}
+
+	return Math.random();
+}
+
+/**
+ * Generates random integers a-la NumPy's in the "half-open" interval [min, max). In other words, from min inclusive to max exclusive. When max is undefined, as is the case by default, results are chosen from [0, min). An error is thrown if max is less than min.
+ *
+ * @param {number} [min = 0] - lowest integer to be drawn, or highest plus one if max is undefined (default)
+ * @param {number} max - one above the largest integer to be drawn
+ * @param {number} [size = 1] - number of values to return
+ * @returns {number} a random integer in the requested range (signed)
+ */
+export function randint(min, max = null, size = 1)
```

Comment:
Should also raise errors if given non-integer negative range parameters like this:
`randint(-10.1, 1)` will return also a non-integer.